src/main/java/io/github/syst3ms/skriptparser/lang/entries/SectionConfiguration.java

```diff
@@ -0,0 +1,125 @@
+package io.github.syst3ms.skriptparser.lang.entries;
+
+import io.github.syst3ms.skriptparser.file.FileSection;
+import io.github.syst3ms.skriptparser.file.VoidElement;
+import io.github.syst3ms.skriptparser.lang.CodeSection;
+import io.github.syst3ms.skriptparser.log.ErrorType;
+import io.github.syst3ms.skriptparser.log.SkriptLogger;
+import io.github.syst3ms.skriptparser.parsing.ParserState;
+import org.jetbrains.annotations.Nullable;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class SectionConfiguration {
+	@Nullable
```

Comment:
Systems used to do this a lot
```suggestion
public class SectionConfiguration {

	@Nullable
```